ang/afform/afsearchSiteTokens.aff.json

```diff
@@ -0,0 +1,31 @@
+{
+    "type": "search",
+    "title": "Site Tokens",
+    "placement": [],
+    "icon": "",
+    "server_route": "civicrm/admin/sitetoken",
+    "permission": [
+        "edit system workflow message templates"
```

Comment:
This permission makes sense - I'm guessing the search display would wind up with 'administer CiviCRM' by default - which might be OK or might be too restrictive? I think we should probably also define this permission in CRM_Core_Permission as the right permission for create / edit these in general (ie it would also apply to the api)